cinema/models.py

```diff
@@ -57,6 +57,10 @@ class MovieSession(models.Model):
     class Meta:
         ordering = ["-show_time"]
 
+    @property
+    def tickets_available(self) -> int:
+        return self.cinema_hall.capacity - self.tickets.count()
```

Comment:
The `tickets_available` property calculates the available tickets by subtracting the count of tickets from the cinema hall's capacity. Ensure that the `tickets` related name is correctly set in the `Ticket` model's `movie_session` ForeignKey, which it is. However, consider handling potential edge cases, such as when the `tickets` count might not be accurate due to concurrent bookings.